lib/OpenCloud/ObjectStore/Resource/AbstractResource.php

```diff
@@ -19,7 +19,9 @@
 
 use Guzzle\Http\Message\Response;
 use OpenCloud\Common\Base;
+use OpenCloud\Common\Http\Client;
 use OpenCloud\Common\Service\ServiceInterface;
```

Comment:
It's still used at https://github.com/LukasReschke/php-opencloud/blob/7696e62992a8a45db0822dae0ac8a8fbfa5c5561/lib/OpenCloud/ObjectStore/Resource/AbstractResource.php#L86-L89